xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -378,10 +378,8 @@ void cancelEndpointDataWatch(String clusterName, EndpointWatcher watcher) {
 
   /**
    * Starts reporting client load stats to a remote server for the given cluster.
-   *
-   * @param loadStatsStore  a in-memory data store containing loads recorded by gRPC client.
    */
-  void reportClientStats(String clusterName, String serverUri, LoadStatsStore loadStatsStore) {
+  void reportClientStats(String clusterName, String serverUri) {
```

Comment:
I thought you were returning `LoadReportClient`. Did you change the plan?

Reply:
Oops... Forgot to change that. Thanks.